source/reactor/inc/reactor/EventLoopImpl.h

```diff
@@ -0,0 +1,52 @@
+#ifndef Reactor_EventLoopImpl_H
+#define Reactor_EventLoopImpl_H
+
+#include "EventLoop.h"
+
+#include <map>
+#include <memory>
+
+namespace u {
+namespace reactor {
+class EventReaction;
+
+class EventLoopImpl : public EventLoop {
+public:
+
+    /**
+     * @see EventLoop::registerHandlers
+     */
+    bool registerHandlers(int fd, long timeout, const EventHandlers &handlers) override;

```

Comment:
Before C++11, the usage of virtual keyword was recommended, but not required, to tell the users of the API that such method was virtual without them having to go to base class and check it. Since C++11, override does the same job and also allows the compiler to complain if the code is inadvertently ill-formed. 

So do not use virtual but do use override